### ISSUE TITLE
Sortieren und filtern?

### ISSUE THREAD

**Issue by arnese** [okfde/sehrgutachten#4]
Lässt sich die Sortier- und Filterfunktion auch unkompliziert übernehmen?
Interessant fänd ich im ersten Schritt auf jeden Fall eine Sortierfunktion nach Datum. 

Später vielleicht eine Filterfunktion nach Abteilung und eine Sortierfunktion nach Seitenzahl.

**Comment by hejjoe** [comment]
Ja, eine Sortierfunktion nach Datum vs Relevanz wäre echt sehr toll.
